Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -245,6 +245,15 @@ Optional<u8> HexEditor::get_byte(size_t position)
     return {};
 }
 
+Vector<u8> HexEditor::get_selected_bytes()
+{
+    Vector<u8> byte_vector;
```

Comment:
Semantically, a `ByteBuffer` is more what we want to use here.

Also, you know how large you want it to be, so you can call `ensure_capacity()` on the vector/bytebuffer which will allocate once instead of growing the buffer repeatedly.

Reply:
Got it, makes sense!